vcd/resource_vcd_vapp_vm.go

```diff
@@ -203,6 +203,13 @@ func resourceVcdVAppVm() *schema.Resource {
 							Type:        schema.TypeString,
 							Description: "Mac address of network interface",
 						},
+						"adapter_type": {
+							Computed:         true,
+							Optional:         true,
+							Type:             schema.TypeString,
```

Comment:
Please swap lines, we use Type everywhere as first

Reply:
Done.